src/main.cpp

```diff
@@ -159,16 +160,18 @@ std::list<SubCommand> build_commands(const ArgumentParser& common)
 
     // install
     auto& install = commands.emplace_back("install", common, [](const ArgumentParser& cmd) {
-        return cmd::run_install({
-            .profile = parse_profile(cmd.get("--profile")),
-        });
+        return cmd::run_install({ .profile = parse_profile(cmd.get("--profile")), .root = cmd.get("--root") });
```

Comment:
add a `,` after `.root = cmd.get("--root")` to make it better formatted.

Reply:
got it